lib/qbwc/response/item_site_query_rs.rb

```diff
@@ -0,0 +1,90 @@
+module QBWC
+  module Response
+    class ItemSitesQueryRs
+      attr_reader :records
+
+      def initialize(records)
+        @records = records
+      end
+
+      def handle_error(errors, config)
+        errors.each do |error|
+          Persistence::Object.handle_error(config,
+                                           error.merge(context: 'Querying item site query'),
+                                           'inventories',
+                                           error[:request_id])
+        end
+      end
+
+      def process(config = {})
+        return if records.empty?
+        puts 'ItemSitesQueryRs#process'
+
+        receive_configs = config[:receive] || []
+        inventory_params = receive_configs.find { |c| c['inventories'] }
```

Comment:
You may have already looked, but where does the integration generate the config files needed for these Get requests? 

I was having issues where getting ServiceProducts wasn't finding the right config here because I was looking for `serviceproducts` but I had run the get_products endpoint.

I'm wondering how exactly this get request might generate the config and will we need to look for `c['inventorywithsites']` or something like that.
I think it will be clearer when testing, so I'm approving, but something to keep an eye on I think!

Reply:
Thanks for the tip! I had to update the PR to use `c['inventorywithsites'` so that it would find the correct config object. For your question, it generates the config in the quickbooks_desktop_endpoint.rb file when it invoked `s3_settings.setup`. That method will look for the JSON file and create it if its not in the folder it expects